docs/blog.html

```diff
@@ -0,0 +1,97 @@
+<!doctype html>
+<html class="no-js" lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <title>Welcome | oVirt</title>
+  <meta name="description" content="">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
+  <link rel="manifest" href="site.webmanifest">
+  <link rel="apple-touch-icon" href="icon.png">
+
+  <link rel="stylesheet" href="css/styles.css">
+
+  <meta name="theme-color" content="#008585">
+</head>
+
+<body>
+  <!--[if IE]>
+    <p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="https://browsehappy.com/">upgrade your browser</a> to improve your experience and security.</p>
+  <![endif]-->
+<div class="ov-wrapper">
+  <header class="ov-header-main">
+        <button id="nav-toggle" class="nav-toggle"></button>
+    <a href=""  aria-label="home-link" class="ov-brand-main"><img src="images/logo.svg" alt="" /></a>
+<nav class="ov-nav-main nav-collapse">
+  <ul class="ov-nav-main__items menu-items">
+    <li class="ov-nav-main__item"><a class="ov-nav-main__link" href="#">Download</a></li>
+    <li class="ov-nav-main__item dropdown"><a  href="#" class="has-dropdown ov-nav-main__link">Documentation</a>
+      <ul class="ov-nav-main__sub-items sub-menu">
+        <li class="ov-nav-main__sub-item ov-nav-main__sub-item--heading">Documentation</li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">Sub link</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">Sub link</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">Sub link</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">Sub link</a></li>
+        <li class="ov-nav-main__sub-item ov-nav-main__sub-item--heading">Release notes</li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">Sub link</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">Sub link</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">Sub link</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">Sub link</a></li>
+      </ul>
+    </li>
+    <li class="ov-nav-main__item"><a class="ov-nav-main__link" href="#">Developers</a>
+    </li>
+    <li class="ov-nav-main__item"><a class="ov-nav-main__link" href="#">Community</a></li>
+    <li class="ov-nav-main__item"><a class="ov-nav-main__link" href="#">Forum</a></li>
+    <li class="ov-nav-main__item"><a class="ov-nav-main__link" href="blog.html">Blog</a></li>
```

Comment:
```suggestion
    <li class="ov-nav-main__item"><a class="ov-nav-main__link ov-nav-main__link--active" href="blog.html">Blog</a></li>
```
We'll need the styles for it, but this will highlight the current page in the main nav

Reply:
Would you pull those styles from the _header.scss file?

Reply:
ya in the `_header.scss` we'd just add the class `ov-nav-main__link--active` to the styles that are getting applied when you hover over it.